appinfo/info.xml

```diff
@@ -63,7 +63,7 @@ Experience the future of data storage with 5GB of free, decentralized storage on
 
 [Get your free licence here!](https://metaprovide.org/hejbit/start) .
     ]]></description>
-    <version>0.5.4</version>
+    <version>0.5.0</version>
```

Comment:
Is there any reason to update the version like this for this PR?

Reply:
The version should be higher than the lastest that we publish. I think @frozenbanana controls the app versionning on the release process.

Reply:
Agreed, this should only be changed on the final release/deploy. Change has been removed